bin/interpret/analyzefiles.py

```diff
@@ -61,7 +61,7 @@ def plotProfile(sample, outDir, lowessData, cnvData, refArray, chromList):
 def plotChroms(sample, outDir, lowessData, refArray, chromList):
 	graphData = [np.mean(lowessData[refArray['chrom'] == x]) for x in chromList]
 	graphErr = [np.std(lowessData[refArray['chrom'] == x]) for x in chromList]
-	graphErr = [abs(graphData[x] - graphErr[x]) for x in range(len(chromList))]
+	#graphErr = [abs(graphData[x] - graphErr[x]) for x in range(len(chromList))]
```

Comment:
huh, yeah, this doesn't seem necessary. I must have been drawing error bars manually at some point...